rails/app/controllers/auth_controller.rb

```diff
@@ -43,7 +43,7 @@ def oauth_authorize
     if current_user.nil?
       validation = AccessGrant.validate_oauth_authorize(params)
       if (!validation.valid)
-        redirect_to validation.error_redirect
+        redirect_to validation.error_redirect, allow_other_host: true
```

Comment:
I made this change in a number of controllers that use `redirect_to` because otherwise tests that used `redirect_to` were failing with an [UnsafeRedirectError](https://api.rubyonrails.org/classes/ActionController/Redirecting/UnsafeRedirectError.html).

I'm not sure if this is the best option or changes the functionality that we want, though. [Here](https://api.rubyonrails.org/classes/ActionController/Redirecting.html) is documentation about redirecting in Rails.

Reply:
This seems fine with the new Rails 7 defaults.